_development.yml

```diff
@@ -1,9 +1,6 @@
-# For development, when using `jekyll serve`
+
 url: http://localhost:4000
-# For development, when using:
-#   analytics --output=[dir]
-# and running from [dir]:
-#   serve
-data_url: https://s3-us-gov-west-1.amazonaws.com/cg-ff27f612-cd16-4a76-94f8-628986af2158/data/live
 
-#data_dir: /reports
+# Use production data by default.
+# To pull from dev data for home page, uncomment:
+# data_url: http://localhost:4000/fake-data
```

Comment:
By default, a dev environment build will point to production data files. Using locally generated `analytics-reporter` data is more of an edge case during development, and I defer to future devs as to how to best make the Jekyll build process work for that case.